invisible_cities/io/mcinfo_io.py

```diff
@@ -187,8 +187,8 @@ def load_mcsensor_response(file_name: str,
         return read_mcsns_response(h5in, event_range)
 
 
-def read_mcinfo_evt (mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
-                     event_number: int, last_row=0) -> ([tb.Table], [tb.Table], [tb.Table]):
+def read_mcinfo_evt (mctables: (tb.Table, tb.Table, tb.Table, tb.Table), event_number: int, last_row=0,
+                     return_only_hits=False) -> ([tb.Table], [tb.Table], [tb.Table]):
```

Comment:
can you add `bool` for the return_only_hits type?

Reply:
Done.